CHANGELOG.md

```diff
@@ -45,6 +37,10 @@
    - `-redis.max-conn-lifetime` changed to `-redis.max-connection-age`
    - `-redis.max-idle-conns` removed
    - `-redis.wait-on-pool-exhaustion` removed
+* [CHANGE] TLS configuration for gRPC, HTTP and etcd clients is now marked as experimental. These features are not yet fully baked, and we expect possible small breaking changes in Cortex 1.5. #3198
```

Comment:
```suggestion
* [CHANGE] TLS configuration for gRPC, HTTP and etcd clients is now marked as experimental. These features are not yet fully backed, and we expect possible small breaking changes in Cortex 1.5. #3198
```

Reply:
`baked` is correct here, as in "baking the cake". It means they are not finished.

Reply:
:facepalm: yes sorry. need more :coffee:

Reply:
I think `baked` is correct here. It's the sense of "cooked".